rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/integration/ProjectTest.java

```diff
@@ -125,7 +371,948 @@ public void should_get_all_projects_paginated() throws IOException, TException {
                         new HttpEntity<>(null, headers),
                         String.class);
         assertEquals(HttpStatus.OK, response.getStatusCode());
+        TestHelper.checkResponse(response.getBody(), "projects", 1);
+    }
+
+    @Test
+    public void should_get_project_by_id() throws IOException {
+        HttpHeaders headers = getHeaders(port);
+        ResponseEntity<String> response =
+                new TestRestTemplate().exchange("http://localhost:" + port + "/api/projects/p001",
+                        HttpMethod.GET,
+                        new HttpEntity<>(null, headers),
+                        String.class);
+        assertEquals(HttpStatus.OK, response.getStatusCode());
+        JsonNode responseBody = new ObjectMapper().readTree(response.getBody());
+        assertEquals("p001", responseBody.get("id").textValue());
+        assertEquals("Project Alpha", responseBody.get("name").textValue());
+        assertEquals("CUSTOMER", responseBody.get("projectType").textValue());
```

Comment:
Please try to use references instead of hard-coding values.

```suggestion
                new TestRestTemplate().exchange("http://localhost:" + port + "/api/projects/" + project1.getId(),
                        HttpMethod.GET,
                        new HttpEntity<>(null, headers),
                        String.class);
        assertEquals(HttpStatus.OK, response.getStatusCode());
        JsonNode responseBody = new ObjectMapper().readTree(response.getBody());
        assertEquals(project1.getId(), responseBody.get("id").textValue());
        assertEquals("Project Alpha", responseBody.get("name").textValue());
        assertEquals("CUSTOMER", responseBody.get("projectType").textValue());
```